packages/workspace/src/generators/new/new.ts

```diff
@@ -103,11 +104,11 @@ function normalizeOptions(options: Schema): NormalizedSchema {
   // -- my-package@2.0.1
   // -- @scope/package@version
   const match = options.preset.match(
-    /^(?<package>(@.+\/)?[^@]+)(@(?<version>\d+\.\d+\.\d+))?$/
+    /^(?<package>(@.+(\/).)?[^@]+)(@(?<version>.+))?$/
```

Comment:
I updated this in https://github.com/nrwl/nx/pull/16303/files so you should after that one lands.